.github/workflows/ci.yml

```
@@ -0,0 +1,52 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  testing:
+    name: Testing
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      # This prevents cancellation of matrix job runs, if one or more already failed
+      # and let the remaining matrix jobs be executed anyway.
+      fail-fast: false
+
+      matrix:
+        php:
+          - '8.2'
+          - '8.3'
```

Maybe you must add an `if` for CGL on 8.4 to exclude it

```suggestion
          - '8.3'
          - '8.4'
```